manifests/init.pp

```diff
@@ -355,9 +357,12 @@
 
 
 class sensu (
-  $version                        = 'latest',
+  $version                        = 'installed',
   $sensu_plugin_name              = 'sensu-plugin',
-  $sensu_plugin_provider          = undef,
+  $sensu_plugin_provider          = $::osfamily ? {
+    'windows' => 'gem',
+    default   => undef,
+  },
```

Comment:
Gem is embedded on windows, e.g. https://github.com/sensu/sensu-puppet/blob/30fe995/lib/puppet/provider/package/sensu_gem.rb#L16

It seems the parent `gem` provider in puppet has a confinement preventing it from working on Windows.  My guess is sensu_gem is inheriting this.  I'll see if I can override and sensu_gem works.